From a491d72c15dffabab0af40a3a09672fef12bb27c Mon Sep 17 00:00:00 2001
From: Ramanathan Choodamani <quic_rchoodam@quicinc.com>
Date: Wed, 15 Mar 2023 12:01:07 -0700
Subject: [PATCH] hostapd: Discard the logic of invalid eht cap size function

The logic for invalidating eht capabilities size needs to be
checked because of which eht capabilities get set to NULL.

Signed-off-by: Ramanathan Choodamani <quic_rchoodam@quicinc.com>
Signed-off-by: Pradeep Kumar Chitrapu <quic_pradeepc@quicinc.com>
---
 src/ap/ieee802_11_eht.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ap/ieee802_11_eht.c b/src/ap/ieee802_11_eht.c
index 6ebe0f9..183b906 100644
--- a/src/ap/ieee802_11_eht.c
+++ b/src/ap/ieee802_11_eht.c
@@ -346,6 +346,8 @@ static bool ieee80211_invalid_eht_cap_size(enum hostapd_hw_mode mode,
 	size_t cap_len;
 	u16 ppe_thres_hdr;
 
+	return false;
+
 	he_capab = (const struct ieee80211_he_capabilities *) he_cap;
 	he_phy_cap = he_capab->he_phy_capab_info;
 	cap = (struct ieee80211_eht_capabilities *) eht_cap;
